manifests/plugins/mesos.pp

```diff
@@ -35,8 +36,13 @@
     content => template('collectd/plugins/mesos/mesos_collectd.py.erb'),
   }
 
-  collectd::plugins::plugin_common { 'mesos':
-    plugin_file_name     => '10-mesos-master.conf',
-    plugin_template_name => 'mesos/10-mesos-master.conf.erb',
+  collectd::plugin { 'mesos':
+    plugin_name         => 'collectd-mesos',
```

Comment:
This package doesn't exists and should be removed.  RHEL platforms break on this change